manifests/profile/network.pp

```diff
@@ -8,8 +8,16 @@
     ensure => stopped,
     enable => false,
   }
-  exec { 'iptables -F':
-    path => '/sbin',
+
+  #Flush existing rules and save blank ruleset on centos 6
```

Comment:
@binford2k either way, the iptables -F only flushes the rules until the next reboot, so it wasn't doing anything anyway.